app/build.gradle

```diff
@@ -49,6 +55,7 @@ dependencies {
             generalTestDependencies.junit,
             generalTestDependencies.mockitoInline,
             generalTestDependencies.kotlinTest
+    implementation 'com.google.android.material:material:1.0.0'
```

Comment:
please add this to the generalDependencies like the other dependencies

Reply:
done